opencensus/stats/measure.h

```diff
@@ -31,6 +31,30 @@ namespace stats {
 template <typename MeasureT>
 class Measure final {
  public:
+  // Registers a MeasureDescriptor, returning a Measure that can be used to
+  // record values for or create views on that measure. Only one Measure may be
+  // registered under a certain name; subsequent registrations will fail,
+  // returning an invalid measure. Register* functions should only be called by
+  // the owner of a measure--other users should use GetMeasureByName. If there
+  // are multiple competing owners (e.g. for a generic resource such as "RPC
+  // latency" shared between RPC libraries) check whether the measure is
+  // registered with MeasureRegistry::GetMeasure*ByName before registering it.
+  //
+  // 'name' should be a globally unique identifier. It is recommended that this
+  //   be in the format "<domain>/<path>", e.g. "example.com/client/foo_usage".
+  // 'units' are the units of recorded values. The recommended grammar is:
+  //     - Expression = Component { "." Component } {"/" Component }
+  //     - Component = [ PREFIX ] UNIT [ Annotation ] | Annotation | "1"
+  //     - Annotation = "{" NAME "}"
+  //   For example, string “MBy{transmitted}/ms” stands for megabytes per
```

Comment:
Cribbing from https://github.com/census-instrumentation/opencensus-specs/blob/afbb08857d94a7876ff69ec0b34e596da72f5c49/stats/gRPC.md

Latencies are measures in milliseconds, denoted "ms"
Sizes are measured in bytes, denoted "By"
Dimensionless values have unit "1"

Reply:
Added.